Docs/pipelines.md

```diff
@@ -104,16 +104,46 @@ Supported Version inputs are:
+| **AppServicePlanPythonName**                  | User / Create Shared Resources                                                                                | Name of the App Service Plan for Python bots.                                                                                                                    | -               | appservicepython                   |
+| **ResourceGroup**                             | User                                                                                                          | (optional) Name of the Resource Group where the bots are deployed.                                                                                               |                 | bots-group                         |
+| **ResourceSuffix**                            | Create Shared Resources                                                                                       | (optional) Suffix to add to the resources' name to avoid collitions (use lowercase, only letters and numbers).                                                   |                 | suffix                             |
+| **[BotName](./availableBotsList.md) + AppId** | [App Registration Portal](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) | (optional) Skill Bot App ID to use. If not configured, it will be retrieved from the key vault.                                                                  |                 | x0x000x-000x-000x-xxx0-x00x0000xxx |
+| **SharedResourceGroup**                       | Create Shared Resources                                                                                       | (optional) Name of the Shared Resource Group where the shared resources are deployed.                                                                            | **BFFN-Shared** | shared-resource-group              |
+
+### Dependency Variables
+
+The following parameters will be displayed in the run pipeline blade.
+
+The version parameters can be set here or you can create variables to set the values for the next pipeline's runs.
+Supported values are: LATEST (default), STABLE, or a specific version.
+For example:
+DEPENDENCIESVERSIONDOTNET = 4.13.1
+
+![dependenciesParameters](./media/dependenciesParametersTest.png)
+
+| Parameter Name                | Source | Description                                                                   | Default value | Example |
+| ----------------------------- | ------ | ----------------------------------------------------------------------------- | ------------- | ------- |
+| **DependenciesVersionDotNet** | User   | (optional) Source from which the Bot Builder dependencies will be downloaded. | -             | NuGet   |
```

Comment:
Isn't the default value Artifacts?

Reply:
This variable was reused from the Deploy pipeline section, this variable default value is empty, it doesn't have a default value, since it's only used for the pipeline's parameter, which the default value is Artifacts, if we want to add the default value to be Artifacts, we should do it in the Deploy doc as well.